README.md

```diff
@@ -72,87 +72,88 @@ Thank you to the people who have already contributed:
 
 Here is a list of the validators currently available.
 
-Validator                               | Description
```

Comment:
For better diffing, just change the section involving isIBAN as opposed to reformatting the whole README.